index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>VidiTube</title>
+    <link href="stylesheet.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="header-section">
+        <header>
+            <div id="logo">
+                <img src="./images/logo.PNG" alt="VidiTube Logo" width="60" height="60" class="image-logo"/>
+                <span class="name">VidiTube</span>
+            </div>
```

Comment:
- [ ] You should use the `a` tag to wrap your logo and give it an attribute of `href="/"` for semantic reasons so a user can navigate through the webpage by clicking on the logo to get back to the homepage.